packages/fast-usdc/src/type-guards.js

```diff
@@ -93,3 +93,24 @@ export const PoolMetricsShape = {
   totalRepays: AmountShape,
 };
 harden(PoolMetricsShape);
+
+/** @type {TypedPattern<ChainPolicy>} */
+export const ChainPoliciesShape = {
+  nobleContractAddress: EvmHashShape,
+  cctpTokenMessengerAddress: EvmHashShape,
+  confirmations: M.number(),
+  chainName: M.string(),
+  bridgeContractAddress: M.opt(EvmHashShape),
+  eventFilter: M.opt(M.string()),
+  chainType: M.opt(M.number()),
+  isL2: M.opt(M.boolean()),
+};
```

Comment:
```suggestion
export const ChainPoliciesShape = M.splitRecord({
  nobleContractAddress: EvmHashShape,
  cctpTokenMessengerAddress: EvmHashShape,
  confirmations: M.number(),
  chainName: M.string(),
  },{
  bridgeContractAddress: EvmHashShape,
  eventFilter: M.string(),
  chainType: M.number(),
  isL2: M.boolean(),
});
```

That way you don't need `eventFilter: undefined`. You can just leave the property out.

Reply:
Done thanks